3rdparty/tools/act/BUILD

```diff
@@ -40,7 +40,7 @@ shell_command(
     command="tar -xf act.tar.gz",
     tools=["tar", "gzip"],
     execution_dependencies=[":downloaded-act"],
-    output_directories=["act"],
+    output_files=["act"],
```

Comment:
The output checking support in this PR flagged this line as an error since it did not detect a directory at that location since it is a file.